Dockerfile

```diff
@@ -1,20 +1,27 @@
-FROM ubuntu:14.04
-MAINTAINER Whit Morriss <whit.morriss@canonical.com>
+FROM ubuntu:16.04
+MAINTAINER Matthew Bruzek <Matthew.Bruzek@canonical.com>
 
-ADD setup.sh /setup.sh
-RUN /setup.sh
+RUN useradd -m ubuntu -s /bin/bash
 
-RUN mkdir /home/ubuntu/.juju
-RUN mkdir /home/ubuntu/trusty
-RUN mkdir /home/ubuntu/precise
+RUN mkdir -p \
+    /home/ubuntu/.local/share/juju \
+    /home/ubuntu/builds \
+    /home/ubuntu/trusty \
+    /home/ubuntu/xenial
 
-VOLUME ["/home/ubuntu/.juju"]
+RUN chown -R ubuntu:ubuntu /home/ubuntu
 
-#ADD patchcontainer.sh /patchcontainer.sh
+VOLUME [ \
+    "/home/ubuntu/.local/share/juju", \
+    "/home/ubuntu/builds", \
+    "/home/ubuntu/trusty", \
+    "/home/ubuntu/xenial" \
```

Comment:
jujubox does not install charm-tools. I removed that with this pull request. Is that a problem for anyone? I want to keep the image as small as possible, and charmbox seems like the right place to install charm-tools.